GCP/deploy.sh

```diff
@@ -2,78 +2,100 @@
 set -e
 
 # Timing doesn't work on zsh currently (works on bash)
-START_TIME=`date "+%s"`
+START_TIME=$(date "+%s")
 
 # Put same user as in ssh key here
-user=dave
+user=julien.sudan
 # If set to true, your existing kubectl config will be OVERWRITTEN, if set to false, script will only download kubectl config for you
 load_kube_config=true
 
 #-----
 # Below ZONE and PROJECT variables are just in case You don't have those defaults set up in gcloud (you should).
 ZONE="europe-west1-b"
 PROJECT="default"
-DEFAULT_PROJECT=`gcloud config list --format 'value(core.project)'`
-DEFAULT_ZONE=`gcloud config list --format 'value(compute.zone)'`
+DEFAULT_PROJECT=$(gcloud config list --format 'value(core.project)')
+DEFAULT_ZONE=$(gcloud config list --format 'value(compute.zone)')
 ZONE="${DEFAULT_ZONE:-$ZONE}"
 PROJECT="${DEFAULT_PROJECT:-$PROJECT}"
 CLUSTER_NAME="${1:-k3s}"
+FW_RULE_NAME="allow-tcp-6443"
+
+#Put your ssh private key path here if not default
+privkeypath="/home/nedsi/.ssh/julien-rsa"
```

Comment:
We should probably remove value from here. Leave it empty so by default `default_privkeypath` is used

Reply:
Sure, it's a mistake, thanks.